webhooks/virtualmachine/validation/virtualmachine_validator.go

```diff
@@ -962,6 +962,7 @@ func (v validator) validateImmutableFields(ctx *context.WebhookRequestContext, v
 	allErrs = append(allErrs, validation.ValidateImmutableField(vm.Spec.ImageName, oldVM.Spec.ImageName, specPath.Child("imageName"))...)
 	allErrs = append(allErrs, validation.ValidateImmutableField(vm.Spec.ClassName, oldVM.Spec.ClassName, specPath.Child("className"))...)
 	allErrs = append(allErrs, validation.ValidateImmutableField(vm.Spec.StorageClass, oldVM.Spec.StorageClass, specPath.Child("storageClass"))...)
+	allErrs = append(allErrs, validation.ValidateImmutableField(vm.Spec.BiosUUID, oldVM.Spec.BiosUUID, specPath.Child("biosUUID"))...)
```

Comment:
A couple of questions about validating the `vm.spec.biosUUID` field:
- Will we add it for old VMs during the upgrade process? If so, this check would always fail the upgrade requests.
- Do we allow DevOps users to create a VM with a specified biosUUID? I don't see it mentioned in the API or enforced in the validation webhook, but I remember that being part of your design.

Reply:
Thanks @dilyar85 , both should be fixed now